server/sockets/logout.js

```diff
@@ -1,5 +1,5 @@
-const { NEW_EVENT_LOGOUT, NEW_EVENT_INACTIVE } = require("../../constants.json");
-const { getActiveUser, delWaitingUser } = require("../utils/lib");
+const { NEW_EVENT_LOGOUT, NEW_EVENT_INACTIVE, NEW_EVENT_CLOSE } = require("../../constants.json");
+const { getActiveUser, delWaitingUser, closeChat } = require("../utils/lib");
```

Comment:
why did you import closeChat here

Reply:
Because, it was used on line 18 and not imported.

Reply:
if it was used and not imported we would have got an error a long time ago, remove the change for that line, I suspect it to be the problem also it has nothing to do with your PR

Reply:
hmmmm, tough spot